examples/connext_dds/README.md

```diff
@@ -0,0 +1,22 @@
+# Connext DDS
+
+RTI Connext DDS is a software connectivity framework for real-time distributed
+applications. It provides the middleware communications service that
+programmers need to distribute time-critical data between embedded and/or
+enterprise devices or nodes. Connext DDS uses a publish-subscribe
+communications model to make data-distribution efficient and robust. Connext
+DDSalso supports the Request-Reply communication pattern1.
+
+Connext DDS implements the Data-Centric Publish-Subscribe (DCPS) API of the
+OMG’s specification, Data Distribution Service (DDS) for Real-Time Systems. DDS
+is the first standard developed for the needs of real-time systems, providing
+an efficient way to transfer data in a distributed system. With Connext DDS,
+you begin your development with a fault-tolerant and flexible communications
+infrastructure that will work over a wide variety of computer hardware,
+operating systems, languages, and networking transport protocols. Connext DDS
+is highly configurable, so programmers can adapt it to meet an application’s
+specific communication requirements.
+
+In this directory you will find some examples about Connext DDS. For more info
```

Comment:
In this directory you will find examples that illustrate some of the most important features of the RTI Connext DDS core libraries. For more information out, please refer to the [RTI Connext DDS User's Manual](link...)